### ISSUE TITLE
Add Java support

### ISSUE THREAD

**Issue by idoo** [houndci/hound#881]

**Comment by idoo** [comment]
http://findbugs.sourceforge.net/
https://pmd.github.io/
http://checkstyle.sourceforge.net/
http://www.jutils.com/

**Comment by gylaz** [comment]
We don't have plans to support Java at this time.

**Comment by sean0x42** [comment]
Is there a technical reason for Hound to not support Java? Why wouldn't you want to expand Hound to cover as many languages as possible?

**Comment by salbertson** [comment]
@sean0x42 nope, no technical reason at all. I'm reopening this since it has been a couple years.

**Comment by sirisak1985** [comment]
f
